src/AerialShip/SamlSPBundle/DependencyInjection/Security/Factory/SamlSpFactory.php

```diff
@@ -181,10 +181,22 @@ protected function createServices()
         }
 
         $services = array();
-        // load db configuration, NOTE: This is a hack!
-        $documentRoot = str_replace('web', '', $_SERVER['DOCUMENT_ROOT']);
-        $file = file_get_contents($documentRoot.'/app/config/parameters.yml');
-        $configParams = Yaml::parse($file);
+        // load db configuration, NOTE: This is a hack! (made hackier by Mike M)
+        if(isset($_SERVER['SYMFONY__DB__HOST']) && isset($_SERVER['SYMFONY__DB__USER']) && isset($_SERVER['SYMFONY__DB__PASSWORD']) && isset($_SERVER['APPLICATOIN_ENV']) && isset($_SERVER['SYMFONY__SAML__KEY__PASS'])) {
```

Comment:
`isset()` can take multiple arguments and will only return true if all of the arguments are set, it can shorten the line